packages/orama/src/utils.ts

```diff
@@ -14,10 +11,11 @@ const second = BigInt(1e9)
 export const isServer = typeof window === 'undefined'
 
 export function sprintf(template: string, ...args: (string | number)[]): string {
-  return template.replaceAll(
+  return template.replace(
```

Comment:
What problem did you identify in this?

Reply:
Oh, the replaceAll method was introduced on Node [v15](https://github.com/nodejs/node/releases/tag/v15.0.0) with the V8 8.5 engine..